x-pack/legacy/plugins/epm/common/types.ts

```diff
@@ -99,6 +99,15 @@ export type ElasticsearchAssetTypeToParts = Record<
   ElasticsearchAssetParts[]
 >;
 
+export interface DataSet {
```

Comment:
@neptunian I had to introduce this to be able to filter on datasets. This might become useful for you.